### DISCUSSION TITLE
子节点移入父节点  node:change:parent触发两次的原因是什么

### DISCUSSION THREAD

**Discussion by avrinfly** [antvis/X6#1842] [Q&A]
把一个子节点移入一个父节点，会触发两次node:change:parent方法，请问有人知道是为什么吗？另外，透传的options里的值具体是啥含义，有人知道吗？

![image](https://user-images.githubusercontent.com/30068005/155504107-390df1f0-4d28-4bcd-9687-75885ba66afa.png)

![image](https://user-images.githubusercontent.com/30068005/155504061-51ba95d0-aae1-4869-8a8b-2e7272e65008.png)

**Comment by NewByVector** [comment]
可以弄个 codesanbox demo，方便问题定位。
